### PR TITLE
Auton map waypoints now display WaypointName,index

### PR DESCRIPTION
**Pull request by jnnanni**
image

### CODE REVIEW

**Comment by CameronTressler** [comment]
Is one able to set the name of the waypoint in the waypoint editor on the GUI? I'm a bit confused since it doesn't seem like there's any code to allow users to set it but I may be missing something

File: base_station/gui/src/components/RoverMapAuton.vue [hunk 1]
```diff
@@ -7,11 +7,11 @@
       <l-tile-layer :url="url" :attribution="attribution" :options="tileLayerOptions"/>
       <l-marker ref="rover" :lat-lng="odomLatLng" :icon="locationIcon"/>
       <l-marker :lat-lng="waypoint.latLng" :icon="waypointIcon" v-for="(waypoint,index) in route" :key="waypoint.id" >
-        <l-tooltip :options="{ permanent: 'true', direction: 'top'}"> {{ index }} </l-tooltip>
```
Comment (CameronTressler) [inline base_station/gui/src/components/RoverMapAuton.vue:10]: Could you add a space after the comma for styling?

File: base_station/gui/src/components/RoverMapAuton.vue [hunk 2]
```diff
@@ -7,11 +7,11 @@
...
+        <l-tooltip :options="{ permanent: 'true', direction: 'top'}"> {{ waypoint.name }},{{ index }} </l-tooltip>
       </l-marker>
 
       <l-marker :lat-lng="waypoint.latLng" :icon="waypointIcon" v-for="(waypoint,index) in list" :key="waypoint.id">
-         <l-tooltip :options="{ permanent: 'true', direction: 'top'}"> {{ index }} </l-tooltip>
```
Comment (CameronTressler) [inline base_station/gui/src/components/RoverMapAuton.vue:14]: Same here

**Review by cgiger00 — APPROVED** [review]

**Comment by cgiger00** [comment]
lgtm

**Review by CameronTressler — APPROVED** [review]
lgtm me if you fix the conflicts in github

**Review by zwgold — APPROVED** [review]

**Pull request merged**